delft3dgtprotectedservices/test_views.py

```diff
@@ -30,22 +34,60 @@ def setUp(self):
         )
         assign_perm("view_scene", self.user_foo, self.scene)
 
+        self.user_foo.user_permissions.add(
```

Comment:
I believe this is not required. We're using an object based permissions model, so we only need to add it once as you do on r40.

Reply:
fixed